CHANGELOG.md

```diff
@@ -10,11 +10,16 @@ ___Note:__ Yet to be released changes appear here._
 
 * `DEPS`: update to bpmn-js-properties-panel@5.2.0
 * `DEPS`: update to camunda-bpmn-js@3.3.1
+* `DEPS`: update to camunda-dmn-js@1.2.0
 
 ### BPMN
 
 * `FEAT`: improve tooltip content ([bpmn-js-properties-panel#955](https://github.com/bpmn-io/bpmn-js-properties-panel/pull/955))
 
+### DMN
+
+* `FEAT`: use FEEL editor in decision table
```

Comment:
Did we miss https://github.com/bpmn-io/dmn-js/issues/768?

Reply:
It's also in the decision table (both input editor and table cell).

Reply:
Let's link these dmn-js items for a more meaningful changelog.

Reply:
Same applies in https://github.com/camunda/camunda-dmn-js/blob/main/CHANGELOG.md#120. Good example here: https://github.com/camunda/camunda-bpmn-js/blob/main/CHANGELOG.md#330.

We'll otherwise hardly be able to keep up with the changes.

Reply:
Check out https://github.com/camunda/camunda-dmn-js/pull/79